pyproject.toml

```diff
@@ -10,18 +10,18 @@ packages = [{include = "pandasai"}]
 [tool.poetry.dependencies]
 python = ">=3.9,<3.9.7 || >3.9.7,<4.0"
 python-dotenv = "^1.0.0"
-pandas = "1.5.3"
+pandas = ">=2.0,<3.0"
```

Comment:
This should allow to install latest pandas 2.2.2 but if you look at the lock file, you will see pandas version = "2.1.4". There are probably some cross-dependencies that are not compatible with pandas 2.2.2. Does anybody know what we can do here to get latest pandas?

Reply:
@gventuri, any thoughts?

Reply:
@gventuri, just a friendly reminder

Reply:
@gventuri, any thoughts?